anda.hcl

```diff
@@ -1 +1,6 @@
+config {
+    strip_prefix = "anda/"
```

Comment:
Would we have to update any of the github workflows, since the prefix has changed?

Reply:
No, these are all simply aliases.